packages/babel-plugin-transform-class-properties/test/fixtures/regression/T2983/actual.js

```diff
@@ -0,0 +1,7 @@
+call(class {
+  static test = true
+});
+
+export default class {
+  static test = true
+};
```

Comment:
I could but in the class expression case the name doesn't really matter, and in the export default case named classes were already working, so I didn't add the tests for those cases before.